integrations/larky/crypto/SHA256_HMAC/README.md

```diff
@@ -0,0 +1,45 @@
+# SHA256 + HMAC sample of encryption in Larky
+
+This sample includes:
+1. Larky test `.star` file that generates the same result due to static input values;
+2. Ready-for-use YAML (Inbound Route) that produces the same result as Larky test.
+
+## Testing part:
+
+#### 1. Larky test:
+
+To be able to run Larky locally, you'll need to setup your local environment:
+https://www.verygoodsecurity.com/docs/larky/test-larky-locally
+
+Example of run:
+
+<img width="1365" alt="image" src="https://user-images.githubusercontent.com/78090218/190900870-7bb2c66c-ec46-41ee-b54a-0463fb174fb8.png">
+
+#### 2. YAML file:
+
+Upload the YAML to your vault and run (secrets here are not real, they were generated by randomizer):
+```
+curl https://tntbmt67sc7.sandbox.verygoodproxy.com/post -k \
```

Comment:
This curl command does not work. If the vault ID does not actually work, we should replace the vault ID with a placeholder to make it clear to the user. For example: `curl https://$VAULT_ID.sandbox.verygoodproxy.com/post -k \`

Reply:
Done